src/transformers/modeling_utils.py

```diff
@@ -551,6 +551,13 @@ def _load_state_dict_into_meta_model(model, state_dict, loaded_state_dict_keys,
     return error_msgs
 
 
+def mask_value(dtype, device):
+    """
+    return a large negative value to be used for attention mask.
+    """
+    return torch.tensor(torch.finfo(dtype).min, dtype=dtype).to(device)
```

Comment:
```suggestion
    return torch.finfo(dtype).min
```
I think returning the value is better here.